sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -1160,6 +1160,22 @@ class DDLParserSuite extends AnalysisTest {
       DropNamespaceStatement(Seq("a", "b", "c"), ifExists = false, cascade = true))
   }
 
+  test("alter namespace set dbproperties") {
+    val sql1 = "ALTER DATABASE a.b.c SET DBPROPERTIES ('a'='a', 'b'='b', 'c'='c')"
+    val sql2 = "ALTER SCHEMA a.b.c SET DBPROPERTIES ('a'='a')"
+    val sql3 = "ALTER NAMESPACE a.b.c SET DBPROPERTIES ('b'='b')"
```

Comment:
Can we have a test case for `PROPERTIES` too since this PR adds it?

Reply:
BTW, shall we follow the following patterns instead of having `val sqlX`?
```scala
    comparePlans(
       parsePlan("DROP NAMESPACE a.b.c CASCADE"),
       DropNamespaceStatement(Seq("a", "b", "c"), ifExists = false, cascade = true))
```